dev/view.js

```diff
@@ -13,8 +13,11 @@ genotet.test = function() {
 
   genotet.viewManager.createView('expression', 'My Expression Matrix', {
     matrixName: 'expressionMatrix',
-    geneRegex: 'sig.*',
-    condRegex: 'SigH.*'
+    dataName: 'B. Subtilis',
```

Comment:
I don't think dataName is defined? matrixName corresponds to the data identifier.